ouroboros-consensus/test-consensus/Test/Dynamic/Network.hs

```diff
@@ -342,6 +343,7 @@ runNodeNetwork registry testBtime numCoreNodes nodeJoinPlan nodeTopology
             , blockMatchesHeader  = nodeBlockMatchesHeader
             , maxUnackTxs         = 1000 -- TODO
             , mempoolCap          = MempoolCapacity 10 -- TODO
+            , mempoolExpThreshold = ExpiryThreshold (NumSlots 1) -- TODO
```

Comment:
1 seems low to me. My intuition says it should at least be greater than the number of core nodes?

Reply:
Yeah, at the time I think I just set it to the lowest value at which the tests seemed to pass. 

Also, just a note that this branch is way behind `master`. I'll need to `rebase` and get back around to finishing this at some point.

Reply:
I believe we also decided that performing expiration based on slot numbers isn't a great idea. So much of this PR will have to change.

Reply:
I wish you all the boons on your journey.